src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -97,6 +97,27 @@
     @Config.RequiresMcRestart
     public static int gasTurbineBonusOutput = 6144;
 
```

Comment:
We should probably create whole new section for Plasma Turbine in _MachineSpecificConfiguration_ which we can put these in.

Also we could move `plasmaTurbineBonusOutput` to that section which would introduce compatibility issue but we can mitigate it by mentioning it in change log. What you think @pyure ?

Reply:
Sections are good.  Personally I'd leave the old value there and override it with the one one, to preserve compatibility.  Probably not crucial though since this doesn't have the capacity to blow up bases.

Reply:
(I'd edit the old comment to indicate that the field is deprecated / overriden)

Reply:
Sounds good we can do that. Let's add new class `ConfigHelper.java` which will handle checking for both values and if one is not set to default it will be use. Deprecation of used old value can be indicated in `@Config.Comment` with info regarding new place for value. And we can remove old one later.